trackintel/preprocessing/positionfixes.py

```diff
@@ -15,7 +15,7 @@ def generate_staypoints(
     distance_metric="haversine",
     dist_threshold=100,
     time_threshold=5.0,
-    gap_threshold=1e6,
+    gap_threshold=15.0,
```

Comment:
why such a drastic change of value? does it mean previous was wrong?

Reply:
see #245. Previously the function was not "gap aware". This is more of a definition change than a bug.